docs/dev/reference/changelog.md

```diff
@@ -41,13 +41,37 @@ date: "2024-09-18"
 
 <!-- If there is no concrete date for a change that makes sense, use the end of the month it was released in. -->
 
+{{% changelog color="removed" title="Managed Processes" date="2025-02-01" %}}
+
+[Managed Processes](/manage/reference/processes/) are now deprecated and will be removed in a future version of `viam-server`.
```

Comment:
Can we say what people should do instead?

Reply:
And it looks like this changelog addition is also made here https://github.com/viamrobotics/docs/pull/3998